scripts/build_wheel.py

```
@@ -524,9 +570,125 @@ def build_kv_cache_manager_v2(project_dir, venv_python, use_mypyc=False):
+    A single reader/writer pair with kernel-buffered pipe I/O is much faster
+    than per-file copies on network filesystems. Dereferences symlinks (-h)
+    and preserves mtimes, matching copytree(symlinks=False) + copystat.
+    Returns False if tar is unavailable or fails, so callers can fall back.
+    """
+    tar_bin = shutil.which("tar")
+    if tar_bin is None:
+        return False
+    dst.mkdir(parents=True, exist_ok=True)
+    # posix (pax) format keeps sub-second mtimes; the gnu default truncates
+    # to whole seconds, which would defeat sync_tree's mtime comparison.
+    result = run(
```

Ruff (0.16.1)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)